modules/tests/src/test/scala/aecor/tests/ScheduleEventCodecSpec.scala

```diff
@@ -13,7 +14,8 @@ class ScheduleEventCodecSpec extends AkkaSpec with PropertyChecks {
   val decoder = PersistentDecoder[ScheduleEvent]
 
   implicit val arbitraryLocalDateTime = Arbitrary(Gen.lzy(Gen.const(LocalDateTime.now())))
-  implicit val arbitraryInstant = Arbitrary(Gen.lzy(Gen.const(Instant.now())))
+  implicit val arbitraryInstant =
+    Arbitrary(Gen.lzy(Gen.const(Instant.now().`with`(ChronoField.MICRO_OF_SECOND, 0L))))
```

Comment:
https://travis-ci.org/notxcain/aecor/jobs/548042560#L554-L560

OpenJDK 9+ offers a more precise system clock, but `PersistentEncoder[ScheduleEvent]` uses the milliseond only. For backward compability, I just dropped microseconds in the spec. WDYT?
https://bugs.openjdk.java.net/browse/JDK-8068730

Reply:
I'm fine with that. We did the same in an internal branch.

Reply:
We used `.truncatedTo(ChronoUnit.MILLIS)` WDYT?